examples/pytorch/question-answering/run_seq2seq_qa.py

```diff
@@ -422,7 +420,7 @@ def preprocess_sqaud_batch(
         answers = examples[answer_column]
 
         def generate_input(_question, _context):
-            return " ".join(["question:", _question, "context:", _context])
+            return " ".join(["question:", _question.lstrip(), "context:", _context.lstrip()])
```

Comment:
I see this function is called `preprocess_sqaud_batch` => can you rename it to `preprocess_squad_batch` please.